covalent/_serialize/lattice.py

```diff
@@ -0,0 +1,242 @@
+# Copyright 2021 Agnostiq Inc.
+#
+# This file is part of Covalent.
+#
+# Licensed under the GNU Affero General Public License 3.0 (the "License").
+# A copy of the License may be obtained with this software package or at
+#
+#      https://www.gnu.org/licenses/agpl-3.0.en.html
+#
+# Use of this file is prohibited except in compliance with the License. Any
+# modifications or derivative works of this file must retain this copyright
+# notice, and modified files must contain a notice indicating that they have
+# been altered from the originals.
+#
+# Covalent is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE. See the License for more details.
+#
+# Relief from the License may be granted by purchasing a commercial license.
+
+"""Functions to convert lattice -> LatticeSchema"""
```

Comment:
The concerns in this module are very similar to the concerns in the other module:
1. Docstrings are missing.
2. save_assets should probably be called in a loop.
3. Perhaps, just perhaps maybe a more OOP design for the whole _serialize module rather than all these functions. Again @AlejandroEsquivel is the best person to weigh-in here.